### ISSUE TITLE
Mac?

### ISSUE THREAD

**Issue by wallygva** [AcuGIS/Escher#8]
I have installed the Escher plugin on a 2.5 Omeka (classic) site.   Installs OK and I see it as an option on the Dashboard...but when I click the "Escher" box, the page loads as you'd expect but the drop down lists never materialize.   Page just shows only this text on right pane (no addon list to select from):

Select Addon and Click Upload

For more information on addons, see the pages plugins and themes or on Omeka.org.

Platform:   Mac OSX 10.8.5

**Comment by DavidGhedini** [comment]
Hi -
Never tried it on a PC.
Is port 80 open via Apache on your MAC?
David

**Comment by wallygva** [comment]
Yes, it's an older MacPro that serves up a number of websites and several Omeka sites as well.  Typical LAMP stack (though it's Mac OS instead of Linux).  For whatever reason, the Escher plugin fails silently (that is, the part where it should create the dropdown boxes doesn't work) but I find no error's in the system logs.    I haven't looked at the Escher code, is it doing a curl call or something like to build the dropdown?   I suspect it's something low-level like a security setting but was hoping you'd run into this before :)

**Comment by DavidGhedini** [comment]
Sorry - I haven't encountered the issue before and I know exactly nothing about Mac :(

**Comment by Daniel-KM** [comment]
The data are extracted from the omeka addons plugins and themes pages (https://omeka.org/addons) and from https://daniel-km.github.io/UpgradeToOmekaS. I don't have a mac, I can't check the issue.

**Comment by wallygva** [comment]
What should the permissions be on the plugins folder?  world-writeable?  
-- Wally

**Comment by Daniel-KM** [comment]
No, only server writeable. The same than the Omeka log or files.
